examples/connector/java/build.gradle

```diff
@@ -61,10 +61,15 @@ application {
     mainClass = 'connector.JavaConnector'
 }
 
+tasks.register('deleteBuild', Delete) {
```

Comment:
what is this for?

Reply:
this is because when we do a `gradle jar` after making some changes, sometimes gradle doesn't refresh the `build` directory and hence we don't see the new generated files (e.g. for protos). adding this task as a dependency on `copyProtos` will ensure that whenever protos are changed, the build directory is cleared out and gradle will surely create new generated files on running `gradle jar`.

Reply:
Ah ok, that makes sense